polarion/record.py

```diff
@@ -1,5 +1,4 @@
 from enum import Enum
```

Comment:
Something seems to fail in the record.py file.  It fails on the upload of a attachment. When debugging i see that the following line in the test does not work. The status remains blocked (in which state you cannot add an attachment, hence why it fails there)
```
self.executing_test_record.setTestStepResult(step_index, Record.ResultType.PASSED, 'See attachment')
```
The record.py file is not covered much so there is no test for setting the test result yet.
I've added the first bit of the trace here:

```

raceback (most recent call last):
  File "C:\Users\Jesper\SynologyDrive\Relek\tools\python-polarion\tests\test_polarion_test_record.py", line 93, in test_attachment_test_step
    self.executing_test_record.addAttachmentToTestStep(step_index, src_1, 'Test image 1')
  File "C:\Users\Jesper\SynologyDrive\Relek\tools\python-polarion\polarion\record.py", line 294, in addAttachmentToTestStep
    service.addAttachmentToTestStep(self._test_run.uri, self._index, step_index, file_name, title, file_content.read())
  File "C:\Users\Jesper\SynologyDrive\Relek\tools\polarion_tests\venv38\lib\site-packages\zeep\proxy.py", line 46, in __call__
    return self._proxy._binding.send(
```

Reply:
I will have a look at it next week.

Reply:
I was not able to run the test record tests on my Polarion installation since there was an issue downloading the test record attachments (HTTP 401 returned). However, I'd expect that this fail is not due to my changes.